kybra/compiler/kybra_generate/src/source_map/mod.rs

```diff
@@ -323,11 +323,19 @@ impl SourceMap {
                 break; // There should be nothing left to see
             }
         }
+        if end < start {
+            eprintln!(
```

Comment:
This will show to the developer even without verbose won't it? And either way I think we should avoid adding things like this